lib/global-styles.php

```diff
@@ -140,14 +140,7 @@ function_exists( 'gutenberg_is_edit_site_page' ) &&
 
 	// Copied from get_block_editor_settings() at wordpress-develop/block-editor.php.
 	$settings['__experimentalFeatures'] = $consolidated->get_settings();
-	if ( isset( $settings['__experimentalFeatures']['color']['palette'] ) ) {
```

Comment:
Leaving a comment here to point to [this thread](https://github.com/WordPress/gutenberg/pull/32358#discussion_r644234638).